src/main/scala/org/neo4j/spark/util/Neo4jUtil.scala

```diff
@@ -53,7 +55,8 @@ object Neo4jUtil {
         case _ => autoCloseable.close()
       }
     } catch {
-      case _ => throw new Exception("This exception should be logged") // @todo Log
+      case t: Throwable => if (logger != null) logger
+        .error(s"Cannot close ${autoCloseable.getClass.getSimpleName} because of the following exception:", t)
```

Comment:
this should not be an error, at most a warning

Reply:
Done